apps/studio/components/interfaces/Database/Backups/BackupsList.tsx

```diff
@@ -88,6 +89,14 @@ const BackupsList = () => {
                 title="You need additional permissions to trigger a scheduled backup"
               />
             )}
+            <Alert_Shadcn_ variant="default">
```

Comment:
minor: maybe lets bring this into its own component, so scheduled and PITR can use the same component